.github/workflows/reusable-windows-ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Get minimum supported version of CMake
         uses: lukka/get-cmake@latest
         with:
-          cmakeVersion: '3.16.3'
+          cmakeVersion: '3.22.1'
```

Comment:
```suggestion
          cmakeVersion: '3.22.6'
```

Reply:
As it has been discussed internally, this PR builds Fast CDR v1.1.x so it is not required to upgrade the CMake. I have dropped the commit.